crates/ursa/src/ursa/mod.rs

```diff
@@ -2,19 +2,15 @@ mod rpc_commands;
 
 use crate::config::{UrsaConfig, DEFAULT_CONFIG_PATH_STR};
 use rpc_commands::RpcCommands;
-use std::{
-    cell::RefCell,
-    fs::File,
-    io::{prelude::*, Result},
-    path::{Path, PathBuf},
-    process,
-    sync::{
-        atomic::{AtomicBool, AtomicUsize, Ordering},
-        Arc,
-    },
-    thread,
-    time::Duration,
-};
+use std::cell::RefCell;
+use std::fs::File;
+use std::io::{prelude::*, Result};
+use std::path::Path;
+use std::path::PathBuf;
+use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
+use std::sync::Arc;
+use std::time::Duration;
+use std::{process, thread};
```

Comment:
Why flatten imports everywhere?

Reply:
Hi, I am `intellij-rust` i sometimes like to flatten imports...

Reply:
[a8d5523](https://github.com/fleek-network/ursa/pull/65/commits/a8d552335983b7149f8783bb2279d7a1e5533260)